eng/versioning/external_dependencies.txt

```diff
@@ -66,15 +66,15 @@ io.netty:netty-tcnative-boringssl-static;2.0.54.Final
 io.netty:netty-transport-native-epoll;4.1.84.Final
 io.netty:netty-transport-native-unix-common;4.1.84.Final
 io.netty:netty-transport-native-kqueue;4.1.84.Final
-io.projectreactor.netty:reactor-netty;1.1.0-RC1
-io.projectreactor.netty:reactor-netty-http;1.1.0-RC1
-io.projectreactor:reactor-core;3.5.0-RC1
+io.projectreactor.netty:reactor-netty;1.1.0
+io.projectreactor.netty:reactor-netty-http;1.1.0
+io.projectreactor:reactor-core;3.5.0
 io.reactivex:rxjava;1.3.8
 io.vertx:vertx-codegen;4.3.3
 io.vertx:vertx-core;4.3.3
-jakarta.servlet:jakarta.servlet-api;5.0.0
+jakarta.servlet:jakarta.servlet-api;6.0.0
```

Comment:
Do you know why this is a major version upgrade?

Reply:
Upgrade to Jakarta EE 10, https://github.com/spring-projects/spring-boot/wiki/Spring-Boot-3.0.0-RC2-Release-Notes#jakarta-ee-10